scripts/test-bfb-install.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
```

Comment:
Better not to add the test code into the rshim repo.

Reply:
Removed.